wolfssl/src/ssl.rs

```diff
@@ -77,6 +77,88 @@ impl<IOCB: IOCallbacks> SessionConfig<IOCB> {
         }
     }
 
+    /// call `func` on `Self`
+    pub fn apply<F>(self, func: F) -> Self
```

Comment:
Did not get what is the use of this helper. Can you please add an example ?

Reply:
Not sure why I added this one (it was quite an old branch I had lying around) and this one indeed isn't all that helpful or used, I'll drop